changelog/unreleased/9253

```diff
@@ -0,0 +1,8 @@
+Bugfix: Make first sync start fast again. 
```

Comment:
While change logs are awesome, bugfixes for regressions that where never released might bloat it a bit.
The bug was only introduced recently on master and never part of any released version.

Reply:
True. Sorry. Was late already.

Reply:
You are right the visibility of the bar is not depending on well progress but on `overallString.isEmpty()`